sklearn/metrics/_ranking.py

```diff
@@ -44,6 +45,12 @@
 )
 
 
+@validate_params(
+    {
+        "x": ["array-like"],
+        "y": ["array-like"],
+    }
+)
 def auc(x, y):
```

Comment:
The documentation below is not correct. Could you change it to reflect that we accept "array-like" instead of array:

```
    x : array-like of shape (n,)
        X coordinates. These must be either monotonic increasing or monotonic
        decreasing.

    y : array-like of shape (n,)
        Y coordinates.
```

Reply:
updated the documentation in the new commit below